src/components/editMeasure/details/MeasureMetaDataRow.tsx

```diff
@@ -40,47 +28,42 @@ const MeasureMetaDataRow = (props: MeasureMetaDataRowProps) => {
         <td>{description}</td>
         {id && canEdit && (
           <td style={{ width: 160 }}>
-            <button
-              className="action-button"
-              onClick={(e) => {
-                handleOpen(id, e);
-              }}
-              tw="text-blue-600 hover:text-blue-900"
-              data-testid={`select-action-${id}`}
-              aria-label={`select-action-${id}`}
+            <Tooltip
+              data-testid="delete-tooltip"
+              title={"Delete"}
+              placement={"top"}
+              arrow
+            >
+              <DeleteOutlinedIcon
+                onClick={(e) => {
+                  e.preventDefault();
+                  handleClick(id, "delete");
+                }}
+                data-testid={`delete-measure-reference-${id}`}
+                style={{ cursor: "pointer", marginRight: "8px" }}
+                sx={{ color: red[500] }}
+              />
+            </Tooltip>
+
+            <Tooltip
+              data-testid="edit-tooltip"
+              title={"Edit"}
+              placement={"top"}
+              arrow
             >
-              <div className="action">Select</div>
-              <div className="chevron-container">
-                <ExpandMoreIcon />
-              </div>
-            </button>
+              <EditNoteIcon
```

Comment:
can we use same icon that we are using other places-  edit parameters, include, definition, etc in CQL UI builder